apps/editor/src/app/panel/authorCheckPicker.tsx

```diff
@@ -50,7 +50,8 @@ export function AuthorCheckPicker({list, disabled, onChange}: AuthorCheckPickerP
       variables: {
         input: {
           name: authorsFilter,
-          slug: slugify(authorsFilter)
+          slug: slugify(authorsFilter),
+          bio: []
```

Comment:
Should be done on the API level, not the frontend

Reply:
Make optional field or add default?

Reply:
Make it optional